script1

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Ask user for n
+read -p "Enter the value of n: " n
+
+# If both n and number of arguments equal to 2, print only the second parameter and exit
+if test "$n" -eq 2 && test "$#" -eq 2; then
+    echo "The second parameter is '$2'."
+    exit 0
+fi
+
+# If n != number of arguments, check if the n-th parameter exists
+if test "$#" -ge "$n"; then
```

Comment:
Add a comment here saying when this condition is met: `The number of arguments is greater or equal n.`